tests/unit/convert_hl7v2_fhir/controllers/er7/test_er7_extractor.py

```diff
@@ -65,16 +63,16 @@ def test_er7_extractor__missing_nhs_number():
 def test_er7_extractor__missing_segment():
     extractor = ER7Extractor(er7_message=parse_message(RAW_ER7_MESSAGE_MISSING_SEGMENT))
 
-    with pytest.raises(ER7ExtractorError):
+    with pytest.raises(ValidationError):
         extractor.patient_location()
 
-    with pytest.raises(MissingPatientClassError):
+    with pytest.raises(ValidationError):
         extractor.patient_class()
 
-    with pytest.raises(MissingAdmissionTypeError):
+    with pytest.raises(ValidationError):
         extractor.admission_type()
 
-    with pytest.raises(MissingTimeOfAdmissionError):
+    with pytest.raises(ValidationError):
```

Comment:
Ordinarily I'd prefer these to be separate tests so that one of them failing couldn't mask later ones also being broken, but I don't feel strongly enough about it to block on that basis at this point.